lib/jdomodels/src/test/java/org/sagebionetworks/repo/model/dbo/dao/DBOTeamDAOImplTest.java

```diff
@@ -643,4 +644,20 @@ private void bindTeamName(String name, Long teamId){
 		// bind this alias
 		principalAliasDAO.bindAliasToPrincipal(alias);
 	}
+
+	@Test
+	public void testGetState() {
+		UserGroup group = new UserGroup();
+		group.setId(userGroupDAO.create(group).toString());
+		teamsToDelete.add(group.getId());
+
+		Team team = new Team();
+		Long id = Long.parseLong(group.getId());
+		team.setId("" +id);
+		team.setCanPublicJoin(true);
+		team.setCanRequestMembership(false);
+		teamDAO.create(team);
+		// Call under test
+		assertEquals(TeamState.PUBLIC, teamDAO.getState(team.getId()));
```

Comment:
We need a test for the case when the team does not exist. Also you should probably have a test for each of the state values (e.g. to test the persistence of the state)